website/src/content/docs/docs/libraries/xml/guide.md

```diff
@@ -144,12 +148,10 @@ Pet:
   properties:
     tags:
       type: "array"
-      xml:
-        name: "ItemsTags"
       items:
         type: string
         xml:
-          name: ItemsName
+          name: tags
```

Comment:
This will still be `ItemsName` because the tag scalar has an encoded name.

Reply:
Unresolved comment - `tag` on  line `123` still has the encoded name `ItemsName`